website/content/docs/integrations/consul/service-mesh.mdx

```diff
@@ -213,7 +237,7 @@ job "countdash" {
       driver = "docker"
 
       env {
-        COUNTING_SERVICE_URL = "http://${NOMAD_UPSTREAM_ADDR_count_api}"
+        COUNTING_SERVICE_URL = "http://count-api.virtual.consul:9001"
```

Comment:
Maybe note that any port can be used here (just guessing)?

Reply:
Good catch. I've fixed that and made it clear as to why you don't need it in the text below this.